### PR TITLE
ASoC: Intel: Add HDMI support to bxt_pcm512x

### PR DESCRIPTION
**Pull request by xiulipan**
Add HDMI support to bxt_pcm512x.

Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>

### CODE REVIEW

**Comment by xiulipan** [comment]
work with https://github.com/thesofproject/soft/pull/80
And SOF GLK branch

File: sound/soc/intel/boards/bxt_pcm512x.c [hunk 1]
```diff
@@ -102,7 +122,62 @@ static const struct snd_soc_ops aif1_ops = {
...
+		.cpu_dai_name = "HDMI3 Pin",
+		.codec_name = "snd-soc-dummy",
+		.codec_dai_name = "snd-soc-dummy-dai",
+		.platform_name = "0000:00:0e.0",
+		.dpcm_playback = 1,
+		.init = NULL,
+		.nonatomic = 1,
+		.dynamic = 1,
+	},
+
```
Comment (keyonjie) [inline sound/soc/intel/boards/bxt_pcm512x.c]: dont need these FE stuffs.

Comment (xiulipan) [reply sound/soc/intel/boards/bxt_pcm512x.c]: done

**Comment by xiulipan** [comment]
@keyonjie 
PR updated.
Yes, that maybe a good idea.

**Comment by xiulipan** [comment]
@lgirdwood  Ping.

**Pull request merged**